models/salesforce__contact_enhanced.sql

```diff
@@ -65,7 +65,7 @@ select
     {% endif %}
 
     {% if var('user_pass_through_columns',[]) != [] %}
-    , {{ var('user_pass_through_columns') | join (", user.")}}
+    , {{ var('user_pass_through_columns') | join (", salesforce_user.")}}
```

Comment:
This was an interesting case! 
 
In reviewing, I thought this would work. But we did notice that Jinja logic doesn't account for the first pass through column in `salesforce_user`. To account for that, we will need to make this change:
```suggestion
    , salesforce_user.{{ var('user_pass_through_columns') | join (", salesforce_user.")}}
```